tests/networkpolicy_test.go

```diff
@@ -181,16 +181,22 @@ func createVMICirros(virtClient kubecli.KubevirtClient, namespace string, labels
 	return vmi
 }
 
-func checkPingSuccess(fromVmi *v1.VirtualMachineInstance, toIp string) {
-	Expect(tests.PingFromVMConsole(fromVmi, toIp, "-w 8")).To(Succeed())
+func checkPingSuccess(fromVmi, toVmi *v1.VirtualMachineInstance) {
+	ExpectWithOffset(1, toVmi.Status.Interfaces[0].IPs).NotTo(BeEmpty())
+	for _, ip := range toVmi.Status.Interfaces[0].IPs {
+		Expect(tests.PingFromVMConsole(fromVmi, ip, "-w 8")).To(Succeed())
+	}
 }
 
-func checkPingFail(fromVmi *v1.VirtualMachineInstance, toIp string) {
-	Eventually(func() error {
-		return tests.PingFromVMConsole(fromVmi, toIp)
-	}, 15*time.Second, time.Second).ShouldNot(Succeed())
+func checkPingFail(fromVmi, toVmi *v1.VirtualMachineInstance) {
+	ExpectWithOffset(1, toVmi.Status.Interfaces[0].IPs).NotTo(BeEmpty())
+	for _, ip := range toVmi.Status.Interfaces[0].IPs {
+		Eventually(func() error {
+			return tests.PingFromVMConsole(fromVmi, ip)
+		}, 15*time.Second, time.Second).ShouldNot(Succeed())
```

Comment:
We are duplicating the check also for failure we have to use Consistently more than Eventually to ensure that it does not start to work eventually.

Reply:
its not right to use Consistently here because it does ping in the beginning and then stop.

as of the 2nd Expect, its to make sure it stop was just one stop suddenly, as discussed on 4045
wdyt?

Reply:
Consistently will repeat ping until the condition does pass, I think this is what we need to use.

Reply:
both can work, but i think the English should be
"eventually ping should fail" 
because we actually know the policy takes time to be applied and at the end would be applied and then"
"eventually ping would fail"

Reply:
changed the lone Expect to Consistently on #4045 as we agreed.
This PR would be just rebased to use that, once #4045 stabilize, since this PR doesnt change this logic, just the surrounding
of it.